### PR TITLE
adding YML templates to localgov repo

### PR DESCRIPTION
**Pull request by LauraHilliger**
created two YML templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,199 +1,36 @@
-# LocalGov Drupal
+# Welcome! 👋👋🏿👋🏽👋🏻👋🏾👋🏼
```
Comment (msayoung) [inline README.md:1]: Did you mean to change this file?
It seems to be a repetition of Readme on https://github.com/localgovdrupal

**Review by msayoung — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LauraHilliger) [reply README.md:1]: ah, @dajbelshaw sent a separate PR 12 days ago with just that commit. We merged it just an hour ago with @aaronhirtenstein – actually we only wanted to merge the YML templates...I think @aaronhirtenstein was going to chat with you about this

**Comment by finnlewis** [comment]
Just discussing in Merge Monday, @msayoung thinks this needs more work.

**Comment by LauraHilliger** [comment]
@msayoung - I believe we can just close this?

**Comment by finnlewis** [comment]
Thanks @LauraHilliger  - closing this in favour of https://github.com/localgovdrupal/localgov/pull/464

**Pull request closed without merging**